snapcraft/tests/plugins/test_qmake.py

```diff
@@ -41,8 +41,8 @@ def setUp(self):
 
         class Options:
             options = []
-            qt_version = 'qt5'
             project_files = []
+            qt_version = 'qt5'
```

Comment:
I'd prefer to avoid this type of noise (moving code around unchanged).

That said, all of this looks good to me.